### ISSUE TITLE
Checking peoples item history is bugged

### ISSUE THREAD

**Issue by anhility** [anhility/BestInSlotRedux#18]
When right-clicking a guildie -> Show history it throws a bug and lock up the addon, forcing a /reload.

**Comment by Meivyn** [comment]
I personally don't have this bug. History is working as intended.
I'm wondering why some have bugs and others are fine.

**Comment by anhility** [comment]
Have you enabled history tracking in the options and then tried to check a guildies BiS history? That's when it bugs out for me.

**Comment by Meivyn** [comment]
Yes, I enabled history to check strings used by it. But it works fine here.

**Comment by anhility** [comment]
Hmm, that's weird. Approx how many people do you have in your Guild BiS list window?

**Comment by Meivyn** [comment]
Actually only 2, and only one have history available.

**Comment by anhility** [comment]
Might be a problem on my end. Will test with a new SavedVariable file and see if it got fixed.

**Comment by anhility** [comment]
A kind soul sent in the following bug report:
***
Date:  2017-11-28 22:41:18
ID: 1

Error occured in: Global
Count: 1
Message: ..\AddOns\BestInSlotRedux\Modules\History.lua line 91:
   bad argument #1 to 'format' (string expected, got nil)
Debug:
   [C]: format()
   BestInSlotRedux\Modules\History.lua:91: ?()
   BestInSlotRedux\Modules\History.lua:117: GetHistoryLabel()
   BestInSlotRedux\Modules\History.lua:192: Draw()
   ...terface\AddOns\BestInSlotRedux\Frames\FramesCore.lua:197: func()
   ...terface\AddOns\BestInSlotRedux\Frames\FramesCore.lua:116: SetFrameSize()
   ...terface\AddOns\BestInSlotRedux\Frames\FramesCore.lua:197: SetMenu()
   ...face\AddOns\BestInSlotRedux\Frames\GuildBiSLists.lua:56: func()
   ..\FrameXML\UIDropDownMenu.lua:791: UIDropDownMenuButton_OnClick()
   [string "*:OnClick"]:1:
      [string "*:OnClick"]:1
Locals:
(*temporary) = "Modified the %s slot. Previous item: %s, new item: %s"
(*temporary) = nil
(*temporary) = "[Knochenpfeife]"
(*temporary) = "[Muschel des Donnerers]"
(*temporary) = "string expected, got nil"
***
Seems that a the function variable doesn't get a proper value at this line:

`91: L["Modified the %s slot. Previous item: %s, new item: %s"]:format(`
->`92: - _G[previtem and previtem.equipSlot or newitem and newitem.equipSlot or "UNKNOWN"],`
`93: - previtem and previtem.link or NONE,`
 `94: - newitem and newitem.link or NONE`

I'll look into it.
